main/install/js/inputShowPwd.js

```diff
@@ -0,0 +1,58 @@
+/*
+*
+* */
+function inputShowPwd(ele) {
+	this.createNewInput = function (oldObject, oType) {
```

Comment:
Codacy found an issue: [Function has a complexity of 5. Maximum allowed is 4.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5720016)